geode-core/src/main/java/org/apache/geode/management/internal/configuration/mutators/ConfigurationManager.java

```diff
@@ -38,5 +38,5 @@
 
   void delete(T config, CacheConfig existing);
 
-  List<? extends RuntimeCacheElement> list(T filterConfig, CacheConfig existing);
+  <R extends RuntimeCacheElement> List<R> list(T filterConfig, CacheConfig existing);
```

Comment:
these seem equivalent...

Reply:
Yes, It's the same. I think I changed it this way just to demonstrate that we don't need to declare R in the class level.